bot/exts/filters/filter_lists.py

```diff
@@ -287,10 +293,58 @@ async def deny_sync(self, ctx: Context) -> None:
         """Syncs both allowlists and denylists with the API."""
         await self._sync_data(ctx)
 
+    @command(name="filter_report")
+    async def force_send_weekly_report(self, ctx: Context) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
+        await self.send_weekly_autoban_report()
```

Comment:
Would it make more sense to send this to the channel the command was invoked in?

Reply:
hah, yes. I even added the `channel` arg to `send_weekly_autoban_report` to support this, I just never updated this line to send ctx.channel

Reply:
[python-discord/bot@`86aaf02` (#2267)](https://github.com/python-discord/bot/pull/2267/commits/86aaf02213eb92a49be6e4b7852e815ddf1c0272)